wasp/apps/clock.py

```diff
@@ -38,6 +38,7 @@ def foreground(self):
         tick callback every second.
         """
         wasp.system.bar.clock = False
+        
```

Comment:
Don't mix non-functional and functional changes in the same patch.

Reply:
this seems to be an artifact from my testing, will make sure to remove those